e2e/.env.example

```
@@ -0,0 +1,3 @@
+GKE_WORKSPACE=
```

```suggestion
E2E_GKE_WORKSPACE=
```
I think this should be added to the example since you're using E2E prefix?